### ISSUE TITLE
Point Rend customization

### ISSUE THREAD

**Issue by freaad** [facebookresearch/detectron2#1678]
## ❓ How to do something using detectron2 point rend

Hello. I am working on the binary classification for my study.
I see the pointrend in the CVPR 2020. 
I am looking for how to separate this work to my pytorch model.
The person who present CVPR 2020 said that I should check the Semantic_seg.py
So I am currently reading it.
I have couple of questions about it.

The point rend is required the trained FPN weights and adds the extra function for FPN.
Because the PointRendSemSegHead class uses the coarse_sem_seg_head.layers.
If we do not trained network we can not get proper coarse_sem_seg_head. If this understanding is wrong please correct me.
Second, If my understanding is wrong please correct me.
1)  Train my network.
2) Train the PointRendSemSegHead.
The input of this network is my trained network's output and targets ( I am still confused what is target about it is it point or ground truth mask)
2-1 During training 
ground truth point coordinate and our point sample network(I am not sure this is the network or something) is calculated by Cross_entropy loss.
3) test
The FPN output passes the our trainined point coordinate network.
I am a little bit confuse about how we actually combine this part with point sampling and semantic mask.

I hope someone who is better understands explain these parts or corrects my understanding. 
Thank you.

**Comment by ppwwyyxx** [comment]
Github issues are for problems about using the code in this repo. We do not provide help in learning how a specific model or algorithm works.
